index.html

```diff
@@ -67,7 +68,219 @@ <h1 class="header-text">
+
+                <section class="built-with">
+
+                    <button id="t" type="submit">
+                        html
+                    </button>
+                    <button id="f" type="submit">
+                        css
+                    </button>
+                    <button id="a" type="submit">
+                        javaScript
+                    </button>
+                </section>
+
+                <section class="prompt">
+                    <button type="submit">See Project</button>
+                </section>
+            </div>
+        </div>
+        <div class="card-3">
+            <img src="./images/Snapshoot-Portfolio.svg" alt="snap">
+            <h1 class="title">
+                Tonic
+            </h1>
+            <div class="sub">
+                <p>CANOPY</p>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="article">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
 
+                <section class="built-with">
+
+                    <button id="m" type="submit">
+                        html
+                    </button>
+                    <button id="q" type="submit">
+                        css
+                    </button>
+                    <button id="v" type="submit">
+                        javaScript
+                    </button>
+                </section>
+                <section class="prompt">
+                    <button type="submit">See Project</button>
+                </section>
+            </div>
+        </div>
+        <div class="card-4">
+            <img src="./images/post.svg" alt="post">
+            <h1 class="title">
+                Multi-Post
+                Stories
+            </h1>
+            <div class="sub">
+                <p>Multi-Post
+                    Stories</p>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+
+            <div class="article">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <section class="built-with">
+
+                    <button id="l" type="submit">
+                        html
+                    </button>
+                    <button id="o" type="submit">
+                        css
+                    </button>
+                    <button id="r" type="submit">
+                        javaScript
+                    </button>
+                </section>
+
+                <section class="prompt">
+                    <button type="submit">See Project</button>
+                </section>
+            </div>
+        </div>
+    </div>
+    <div class="about">
```

Comment:
- [ ] As pointed out earlier, every section of the page should be a section tag instead of a div for semantic and accessibility reasons